cypress/support/step-definitions/split-button-steps.js

```diff
@@ -9,7 +8,9 @@ import {
 import { positionOfElement } from '../helper';
 
 Then('Split Button is expanded', () => {
-  commonButtonPreviewNoIFrameRoot().should('have.length', 5); // 3 expanded buttons, 1 icon button and 1 main button
+  additionalButton(0).should('exist');
```

Comment:
buttons could exist anywhere in a DOM, please add for every `additionalButton`
```suggestion
  additionalButton(0).should('be.visible');
```